doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -745,6 +745,17 @@ The `mz_network_policy_rules` table contains a row for each network policy rule.
 | `address`        | [`text`]   | The address the rule will take action on.                                                              |
 | `direction`      | [`text`]   | The direction of traffic the rule applies to. `ingress` is the only supported direction.               |
 
+## `mz_show_network_policies`
```

Comment:
This view should be undocumented.

Reply:
I don't think I can remove this.
The sqllogic tests is auto-generated and seems to be fighting the script that autogenerates it when this is removed.

Reply:
Sorry, I missed including the workaround, which is to mark it as undocumented:

`<!-- RELATION_SPEC_UNDOCUMENTED mz_internal.mz_show_network_policies -->`

The weird thing is that none of the other `mz_show` relations are, so not sure if there's something else going on here. But this should make the tests happy, at least!

Reply:
They are all at the bottom of the file, thanks for the pointer!